samples/system-test/topics.test.js

```diff
@@ -83,16 +94,16 @@ describe('topics', () => {
   it('should list topics', async () => {
     const output = execSync(`${commandFor('listAllTopics')}`);
     assert.match(output, /Topics:/);
-    assert.match(output, new RegExp(fullTopicNameOne));
+    assert.match(output, new RegExp(fullTopicNameThree));
```

Comment:
For these places where we're not actually using a regex, `assert.include` feels more clear to me